smarts/env/rllib_hiway_env.py

```diff
@@ -75,9 +116,9 @@ def __init__(self, config):
         a = config.worker_index
         b = config.vector_index
         c = (a + b) * (a + b + 1) // 2 + b
-        smarts.core.seed(seed + c)
+        self._seed = seed + c
+        smarts.core.seed(self._seed + c)
```

Comment:
One thing to note from this PR is that the original behaviour which used to result in environment seed diversity no longer works.

Seed used to vary across all environments during a trial but is now the same within a trial. I am unsure if this has to do with how the environment configuration is distributed.

The implication is that this slows down experience gain.